### ISSUE TITLE
Bug? Prompt in the chat bar disappears when previous answer is finished

### ISSUE THREAD

**Issue by ACuzuiocSR** [mckaywrigley/chatbot-ui#1395]
I'm typing a new prompt while the chatbot is generating an answer to a previous one. When the previous answer finishes to generate, my new prompt in the bar disappears and the chat bar refreshes. I lose the contents that I have typed so far and need to start the process again. Not ideal.

**Comment by siavashvj** [comment]
Yes, this is an issue. Need to make it so that the chatbox is not cleared when response is finished. 

On the same note, it's possible to send multiple requests simultaneously, this should be prevented as well.

**Comment by HowieG** [comment]
Bump
